README.md

```diff
@@ -115,6 +115,16 @@ import { NgxStripeModule } from 'ngx-stripe';
 export class AppModule {}
 ```
 
+or using the provider function on bootstrap app config
+
+```ts
+import { provideNgxStripe } from 'ngx-stripe';
+
+bootstrapApplication(AppComponent, {
+  providers: [provideNgxStripe('***your-stripe-publishable-key***')]
```

Comment:
I actually like this name better, but I think in the actual code you name it `provideStripe`, maybe we can change it and make it all `provideNgxStripe`

Reply:
Hey @richnologies, sure 😄, I  wasn't sure about what name to use, I changed it and forgot the README.
I'll make the change 👍🏻

Reply:
Done! 😄